src/main.rs

```diff
@@ -166,15 +173,28 @@ fn inner_main(language: &mut Option<Language>) -> errors::Result<()> {
 
    log::debug!("init done! starting event loop");
 
+   let mut last_window_size: PhysicalSize<u32> = PhysicalSize::default();
+   let mut last_window_position: PhysicalPosition<i32> = PhysicalPosition::default();
```

Comment:
Hm, I'm a bit concerned that using the default PhysicalSize (which I assume should be `(0, 0)`) will result in the window position/size being saved as `(0, 0)` if the window is never moved or resized during the app's runtime, thus never generating a Resized or Moved event. Have you checked that this works as intended?

What I think this should do instead is probe the window for its size and position, and save that in these variables. Either that or read the size and position from `&config().window`.

Reply:
> Hm, I'm a bit concerned that using the default PhysicalSize (which I assume should be `(0, 0)`) will result in the window position/size being saved as `(0, 0)` if the window is never moved or resized during the app's runtime, thus never generating a Resized or Moved event.

Yes, this is what happens. My bad. I fixed it the way you suggested, trying to read from the config, and if that fails, taking it from the window.